src/cmd/cmdHandler.ts

```diff
@@ -165,4 +190,39 @@ export default class CmdHandler {
 
     return proc;
   };
+
+  _askImageNameAndMounts = async (filepath: string): Promise<lib.BuildOptions> => {
+    let fileName = path.basename(filepath);
+    fileName = fileName.replace(path.extname(fileName), "");
+    let imageName = await vscode.window.showInputBox({
+      title: "Image Name",
+      value: fileName
+    });
+
+    let mounts = await vscode.window.showInputBox({
+      title: "Mount Points",
+      placeHolder: "Comma-separated [volume_id:mount_path]"
```

Comment:
This requires users to know the volume id beforehand. It would be easier for users to select a volume from the list.

Check this example as inspiration https://github.com/microsoft/vscode-extension-samples/tree/main/quickinput-sample. This would facilitate a lot the selection of a volume.

Reply:
Fixed by https://github.com/nanovms/ops-vscode/pull/13/commits/069a777d4b321ea237f013316a2bf4153a1ce43e

Reply:
I could select an existing volume but wasn't able to mount it on the unikernel. This is what console output shows after executing `Ops: Run` with a volume selected `args: run,/mnt/d/Go/src/github.com/fabiodmferreira/walk-server/main,--imagename,lorem`. The argument seems to not be added on executing ops.

UUIDs aren't human friendly. It would be nice to add the volume name on the select options.

Reply:
I can see the `volume_id:mount_path` passed correctly:
```
args: run,/home/arknable/Development/Project/nanovms/example/hello/hello,--imagename,hello,--mounts,b2858b75-cd8e-631b-9f90-af0d89524dc8:/home/arknable/Development/Project/nanovms/tmp/mounts/a
```
probably you forgot to select volume id or input the mount path @fabioDMFerreira. anyway I will add volume name also.

the flow is we select volumes then for each selected volume, the extension will ask user to input mount point path. if we don't select any volume, `--mounts` will not be passed to cli. for each selected volume, if we don't input the mount path, that volume will me omitted.

Reply:
I will there is a way to customize quick pick but for now this is the only option provided by vscode afaik.

Reply:
Volume name added by https://github.com/nanovms/ops-vscode/pull/13/commits/f5696ebafb97440b84c6f172bb986556161e7012.

Reply:
mount flag usage fixed by https://github.com/nanovms/ops-vscode/pull/13/commits/f6290dbee98b6c9b78e1a4e4d9b5312ea9dfc0c6

Reply:
You're right, I wasn't selecting the volume. I thought it was enough to click enter on the volumes list, but we need to click on the checkbox. I think this happens because it is a multiselect.